src/components/icons.js

```diff
@@ -10,34 +10,6 @@ import colors from 'src/libs/colors';
 
 export { icon } from '@terra-ui-packages/components';
 
-/**
- * To support accessibility, every icon must be labeled, either:
- * * by placing it next to relevant text and hiding the icon, or
- * * by explicitly setting `aria-label` or `aria-labelledby` on the icon or its interactive container element.
- *
- * This function attempts to determine if an element with the given set of children and properties
- * contains a single icon without a label, so we can do an appropriate resolution.
- *
- * If the container element has an 'aria-label` or `aria-labelledby`, or if the icon itself
- * has `aria-label` or `aria-labelledby`, or if the icon is accompanied by other child elements, then we can
- * assume the icon is labeled and no further label is needed.
- */
-export const containsUnlabelledIcon = ({ children, 'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy }) => {
```

Comment:
This is no longer used. TooltipTrigger uses a similar function that's located in the components package.